### PR TITLE
[ISSUE-871][REFACTOR] Migrate Worker conf to Celeborn Configuration System

### LINKED ISSUE #871: [REFACTOR] Migrate Worker conf to Celeborn Configuration System
### Describe the intention you'd like
A clear and concise description of what you want to do.

/cc @who-need-to-know

/assign @who-can-help-you

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### Related issues.
close #871

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 1]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.rpc.port")
+      .withAlternative("rss.worker.rpc.port")
+      .categories("worker")
+      .doc("")
+      .version("0.2.0")
+      .intConf
+      .createWithDefault(0)
+
+  val WORKER_PUSH_SERVER_PORT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.pushServer.port")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1284]: how about eliminating "server"? "celeborn.worker.push.port" should be clear enough, and consistent w/ "celeborn.worker.rpc.port"

Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala]: Done

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 2]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+  val WORKER_REGISTER_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.register.timeout")
+      .withAlternative("rss.register.worker.timeout")
+      .categories("worker")
+      .doc("Worker register timeout.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("180s")
+
+  val NON_EMPTY_DIR_EXPIRE_DURATION: ConfigEntry[Long] =
```
Comment (FMX) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1319]: `WORKER_NON_EMPTY_DIR_EXPIRE_DURATION`?
Unify the style ?

Comment (FMX) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1319]:
```suggestion
  val WORKER_NON_EMPTY_DIR_EXPIRE_DURATION: ConfigEntry[Long] =
```

Unify the style?

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 3]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.noneEmptyDirExpireDuration")
+      .withAlternative("rss.expire.nonEmptyDir.duration")
+      .categories("worker")
+      .doc("If a non-empty application shuffle data dir have not been operated during le duration time, will mark this application as expired.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1d")
+
+  val WORKING_DIR_NAME: ConfigEntry[String] =
```
Comment (FMX) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1328]: ditto

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 4]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.workingDirName")
+      .withAlternative("rss.worker.workingDirName")
+      .categories("worker")
+      .doc("Worker's working dir path name.")
+      .version("0.2.0")
+      .stringConf
+      .createWithDefault("hadoop/rss-worker/shuffle_data")
+
+  val CLOSE_IDLE_CONNECTIONS: ConfigEntry[Boolean] =
```
Comment (FMX) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: ditto

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 5]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.closeIdleConnections")
+      .withAlternative("rss.worker.closeIdleConnections")
+      .categories("worker")
+      .doc("Whether worker will close idle connections.")
+      .version("0.2.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val REPLICATE_FAST_FAIL_DURATION: ConfigEntry[Long] =
```
Comment (FMX) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1346]: ditto

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala (hunk 2 above)
Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala]: How about current?

**Review by FMX — APPROVED** [review]
LGTM.

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 6]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.disks.check.timeout")
+      .withAlternative("rss.worker.status.check.timeout")
+      .categories("worker")
+      .doc("Timeout time for worker check device status.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("30s")
+
+  val WORKER_CHECK_FILE_CLEAN_MAX_RETRIES: ConfigEntry[Int] =
+    buildConf("celeborn.worker.disks.checkFileClean.maxRetries")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1365]: celeborn.worker.disk

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 7]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+      .doc("Whether worker will close idle connections.")
+      .version("0.2.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val WORKER_REPLICATE_FAST_FAIL_DURATION: ConfigEntry[Long] =
+    buildConf("celeborn.worker.replicate.fastFail.duration")
+      .withAlternative("rss.replicate.fastfail.duration")
+      .categories("worker")
+      .doc("If a relicate request not replied during the duration, worker will mark the replicate data request as failed.")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1350]: typo relicate

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 8]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.noneEmptyDirExpireDuration")
+      .withAlternative("rss.expire.nonEmptyDir.duration")
+      .categories("worker")
+      .doc("If a non-empty application shuffle data dir have not been operated during le duration time, will mark this application as expired.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1d")
+
+  val WORKER_WORKING_DIR_NAME: ConfigEntry[String] =
+    buildConf("celeborn.worker.workingDirName")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1329]: workingDirName => workingDir

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 9]
```diff
@@ -1259,6 +1271,114 @@ object CelebornConf extends Logging {
...
+    buildConf("celeborn.worker.replicate.fastFail.duration")
+      .withAlternative("rss.replicate.fastfail.duration")
+      .categories("worker")
+      .doc("If a relicate request not replied during the duration, worker will mark the replicate data request as failed.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("60s")
+
+  val WORKER_DEVICE_STATUS_CHECK_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.disks.check.timeout")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1356]:
```suggestion
    buildConf("celeborn.worker.disk.check.timeout")
```

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala (hunk 8 above)
Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala]: Done

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala (hunk 7 above)
Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala]: DOne

**Pull request merged**